doctrinal-works-v1/original/11-chapter-9.adoc

```diff
@@ -184,6 +184,7 @@ and to you this is the word of the Lord God,
 from him that dwells in that which comprehends the world, and answering this,
 shall be with that of God in every one of you.
 
+[.signed-section-signature]
 G+++.+++ F.
 
 Have not ribbons, jewels, images and baubles carried away the eyes and hearts of people,
```

Comment:
I think I might make this last little chunk after the signature into a `.postscript`